tests/image_stitching_test.py

```diff
@@ -7,51 +7,63 @@
 
 from toffy import image_stitching, json_utils
 
-RUN_JSON_SPOOF = {
-    "fovs": [
-        {
-            "runOrder": 1,
-            "scanCount": 1,
-            "frameSizePixels": {"width": 32, "height": 32},
-            "name": "R1C3",
-        },
-        {
-            "runOrder": 2,
-            "scanCount": 1,
-            "frameSizePixels": {"width": 16, "height": 16},
-            "name": "R2C1",
-        },
-        {
-            "runOrder": 3,
-            "scanCount": 1,
-            "frameSizePixels": {"width": 8, "height": 8},
-            "name": "MoQC",
-        },
-        {
-            "runOrder": 4,
-            "scanCount": 1,
-            "frameSizePixels": {"width": 16, "height": 16},
-            "name": "R2C2",
-        },
-    ],
-}
-
 
-def make_run_file(tmp_dir):
+def make_run_file(tmp_dir, prefixes=[]):
     """Create a run subir and run json in the provided dir and return the path to this new dir."""
 
+    if len(prefixes) == 1:
+        prefix1, prefix2 = prefixes * 2
+    else:
+        prefix1 = prefixes[0]
+        prefix2 = prefixes[1]
+
+    run_json_spoof = {
```

Comment:
Is there a way we can more programmatically generate this? It's a pretty long test input, maybe consider a loop?

Reply:
Totally, this one kinda got a way from me lol.